<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>x_sql</class>
 <widget class="QDockWidget" name="x_sql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据源</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>已连接列表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <item>
           <property name="text">
            <string>198.8.8.7</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="5" column="3">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>保存密码</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>主机：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>连接名：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>用户名：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>验证：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="6" column="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>密码：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>SQL Server 验证</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Windows 验证</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>测试连接</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
